### PR TITLE
Change inventory total assertions to check http status instead

### PR DESCRIPTION
**Pull request by proubatsis**
Addresses #78

### CODE REVIEW

File: finish/src/test/java/it/io/openliberty/guides/config/ConfigurationTest.java [hunk 1]
```diff
@@ -68,9 +70,8 @@ public void testInitialServiceStatus() {
     boolean status = Boolean.valueOf(ConfigTestUtil.readPropertyValueInFile(
         INV_MAINTENANCE_PROP, DEFAULT_CONFIG_FILE));
     if (!status) {
-      assertEquals("The Inventory Service should be available", 0,
-          ConfigTestUtil.getJsonObjectFromURL(client, baseUrl + INVENTORY_HOSTS, 1,
-              null).getInt("total"));
+        ConfigTestUtil.getJsonObjectFromURL(client, baseUrl + INVENTORY_HOSTS, 1,
+            null, Optional.of(200));
```
Comment (evelinec) [inline finish/src/test/java/it/io/openliberty/guides/config/ConfigurationTest.java]: Can "200" be replaced by "Response.Status.OK"?

File: finish/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java [hunk 2]
```diff
@@ -111,9 +115,15 @@ public static void overwriteFile(String fileName, String newContent) {
...
    */
   public static JsonObject getJsonObjectFromURL(Client client, String url, int level,
-      String key) {
+      String key, Optional<Integer> expectedStatus) {
     Response response = client.target(url).request().get();
 
+    if (expectedStatus.isPresent()) {
+      Integer expected = expectedStatus.get();
+      Integer actual = response.getStatus();
+      assertEquals(expected, actual);
```
Comment (evelinec) [inline finish/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java]: Generally we want to have the assert statement in the actual test case class, so it's easier to tell what are being compared.

File: finish/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java [hunk 3]
```diff
@@ -17,8 +17,8 @@
 import java.io.FileWriter;
 import java.io.FileReader;
 import java.io.IOException;
+
```
Comment (evelinec) [inline finish/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java]: Remove the empty line between imports.

File: start/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java [hunk 4]
```diff
@@ -17,8 +17,8 @@
 import java.io.FileWriter;
 import java.io.FileReader;
 import java.io.IOException;
+
```
Comment (evelinec) [inline start/src/test/java/it/io/openliberty/guides/config/ConfigTestUtil.java]: Remove empty line between imports.

**Comment by proubatsis** [comment]
@evelinec I removed the lines

**Comment by proubatsis** [comment]
I tested the guide this morning and it looks good.

**Comment by evelinec** [comment]
Merge on behalf of @proubatsis.

**Pull request merged**